### DISCUSSION TITLE
Using binary = false checkpoint files.

### DISCUSSION THREAD

**Discussion by Edward-Eth** [idaholab/moose#21334] [Q&A General] [answered]
Hi, I'm working through a seg fault issue when trying to load from checkpoint files, and am attempting to use the binary = false setting in the checkpoint block of my output. This is working sucessfully, and is leading to .cpa and .xda files instead of the normal .cpr and .xdr files. Setting file = xxxx.cpa in mesh is working and the restart is loading the mesh in, however I can't find info on how to tell the problem to search for restart files in the .xda format instead of .xdr, which leads to lots of "unable to find file xxx.xdr" errors when I run. I've tried setting binary = false in problem to mirror the argument given to the checkpoint output, but that doesn't seem to help. Any advice would be appreciated.

**Comment by GiudGiud** [comment, marked as answer]
Hello

I ll have a look into this
In the mean time you can use a SolutionUO
https://mooseframework.inl.gov/source/userobject/SolutionUserObject.html

Guillaume

**Reply by Edward-Eth** [reply]
I'm not sure how well/if that would work when using plasticity, as simply loading displacements in won't correctly recreate the previous state surely?

**Reply by GiudGiud** [reply]
not sure.
loading from xda should work, the code is there and it's been done by others/

Could you please paste your Problem block here
And also the name of the restart folder and its content

**Reply by GiudGiud** [reply]
Any news on this?
